python-sdk/tests/benchmark/infrastructure/kubernetes/job.yaml

```diff
@@ -14,9 +14,9 @@ spec:
         command: ["./run.sh"]
         resources:
           requests:
-            memory: 16Gi
+            memory: 50Gi
```

Comment:
Can we add a comment about why is this high?

Reply:
We can't expect a single worker to have 50Gi in a real-world scenario, this is too much memory IMO

Reply:
why are postgres and snowflake tests failing here?

Reply:
50 GI: this is just wrong... 

I 100% agree with @pankajastro -- we can't keep increasing the worker memory. We should instead fix the issue which is causing us to need that memory

Reply:
We have talked about not changing the benchmarking environment (Datasets, worker size etc) multiple times now

Reply:
@kaxil - You can still run all the benchmarks one at a time in "n2-standard-8" and 16Gi memory. This is a separate issue of memory leak which needs to be investigated. We are only changing this because we are running 150+ benchmarks on the same machine which leads to consuming the memory and not freeing it. WDYT?

Reply:
>We are only changing this because we are running 150+ benchmarks on the same machine which leads to consuming the memory and not freeing it. WDYT?

We should investigate and fix it rather than working around it